src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -110,10 +110,14 @@ public function denormalize($data, string $type, string $format = null, array $c
         $defaultDateTimeFormat = $this->defaultContext[self::FORMAT_KEY] ?? null;
 
         if (null !== $defaultDateTimeFormat) {
-            $object = \DateTime::class === $type ? \DateTime::createFromFormat($defaultDateTimeFormat, $data, $timezone) : \DateTimeImmutable::createFromFormat($defaultDateTimeFormat, $data, $timezone);
-
-            if (false !== $object) {
-                return $object;
+            try {
+                $object = \DateTime::class === $type ? \DateTime::createFromFormat($defaultDateTimeFormat, $data, $timezone) : \DateTimeImmutable::createFromFormat($defaultDateTimeFormat, $data, $timezone);
+
+                if (false !== $object) {
+                    return $object;
+                }
+            } catch (\Throwable $t) {
```

Comment:
Throwable is way too wide, we should be specific here

Reply:
Thank you for your feedback, i have chenged it against Exception